perllib/Open311.pm

```diff
@@ -418,6 +418,21 @@ sub split_name {
     return ( $first, $last );
 }
 
+sub _params_to_string {
+    my( $self, $params, $request_string ) = @_;
+
+    my $undefined;
+
+    my $string = join("\n", map {
+        $undefined .= "$_ undefined\n" unless defined $params->{$_};
+        "$_: " . ( defined $params->{$_} ? $params->{$_} : '' );
```

Comment:
This is fine, but just to note `defined $params->{$_} ? $params->{$_} : ''` could be `$params->{$_} // ''` (`//` is like `||` but tests definedness rather than truth, so e.g. would let through `0`.